From 874dcfbd303bc7a1bddb6f34aebbb4dba8eda771 Mon Sep 17 00:00:00 2001
From: Rafael Siejakowski <rs@rs-math.net>
Date: Sun, 10 Nov 2024 13:20:48 +0100
Subject: [PATCH] Fix Poppler private includes

In the PDF parser, include the Poppler private headers using
the angular brackets and the poppler/ directory prefix, as is
done in other places in the code which need Poppler's private
headers.

This fixes build against Poppler installed at a custom path.
---
 src/extension/internal/pdfinput/pdf-parser.h | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/src/extension/internal/pdfinput/pdf-parser.h b/src/extension/internal/pdfinput/pdf-parser.h
index 8325ea24364..7d9189dfc0a 100644
--- a/src/extension/internal/pdfinput/pdf-parser.h
+++ b/src/extension/internal/pdfinput/pdf-parser.h
@@ -24,13 +24,12 @@
 #pragma interface
 #endif
 
-#include "glib/poppler-features.h"
-#include "Object.h"
-
+#include <2geom/affine.h>
+#include <glib/poppler-features.h>
 #include <map>
 #include <memory>
+#include <poppler/Object.h>
 #include <string>
-#include <2geom/affine.h>
 
 #define Operator Operator_Gfx
 #include <poppler/Gfx.h>
-- 
GitLab

